<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientView</class>
 <widget class="QGroupBox" name="ClientView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Client</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="clientGroup" native="true">
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="lbl">
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nomIn"/>
      </item>
      <item row="1" column="1">
       <spacer name="spc_23">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_2">
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="addrIn"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbl_3">
        <property name="text">
         <string>Telephone</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="madaTelHead">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>+261</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="telIn">
          <property name="inputMask">
           <string notr="true">00 00 000 00</string>
          </property>
          <property name="text">
           <string>00 00 000 00</string>
          </property>
          <property name="maxLength">
           <number>12</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <spacer name="spc_24">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lbl_4">
        <property name="text">
         <string>Sexe</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="sexeIn">
        <item>
         <property name="text">
          <string>Homme</string>
         </property>
         <property name="icon">
          <iconset resource="../../../res/ressources.qrc">
           <normaloff>:/icons/icons/man.svg</normaloff>:/icons/icons/man.svg</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Femme</string>
         </property>
         <property name="icon">
          <iconset resource="../../../res/ressources.qrc">
           <normaloff>:/icons/icons/woman.svg</normaloff>:/icons/icons/woman.svg</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="spc_25">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../res/ressources.qrc"/>
 </resources>
 <connections/>
</ui>
